test/JDBC/input/storedProcedures/BABEL-5059-vu-verify.sql

```diff
@@ -0,0 +1,469 @@
+-- Procedure
```

Comment:
can you also print function signature using `select definition from information_schema.routines  where xyz `? this will ensure that max are handled correctly during DDL export also

Reply:
Done